src/features/invites/RedeemInviteCode.tsx

```diff
@@ -113,6 +113,36 @@ export const RedeemInviteCode = ({
     }
   };
 
+  const updateRedeemedAt = async () => {
+    try {
+      setLoading(true);
+      const { update_profiles_by_pk } = await client.mutate(
+        {
+          update_profiles_by_pk: [
+            {
+              pk_columns: { id: profileId },
+              _set: { invite_code_redeemed_at: 'now()' },
```

Comment:
this is ok for now but this implies anyone can set this, probably not the best pattern